cluster/charts/crossplane/templates/rbac-manager-deployment.yaml

```diff
@@ -74,9 +74,6 @@ spec:
         args:
         - rbac
         - start
-        {{- if .Values.rbacManager.managementPolicy }}
```

Comment:
turns out this condition was also used to gate the creation of the namespaced ClusterRoles [here](https://github.com/negz/crossplane/blob/f6bdc90c4134abde8f7652d02d65e68921d66972/cluster/charts/crossplane/templates/rbac-manager-managed-clusterroles.yaml#L190-L258). @negz, this would be another manual step people would need in order to restore the old behaviour if they cared about the `All` management policy. Should we just drop it and add a warning in the release notes?